blog/2025/01-03 FDM Monster Server release 1.8.2/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: FDM Monster Server release 1.8.2
```

Comment:
Why call it `FDM Monster Server` if RN for FDMM client is included?

Reply:
Fixed!